Data/member.json

```diff
@@ -413,6 +413,13 @@
             "description": "Bangladeshi | Studying Computer Science and Engineering at BRAC University, Dhaka | Webmaster, BRACU IEEE SB | Competitive Programmer |",
             "githuburl": "https://github.com/ZuhairHossain/",
             "linkedinurl": "https://www.linkedin.com/in/syed-zuhair-hossain/"
-        }
+        },
+	    {
```

Comment:
@janzell after you move the bottom closing brace, please move the whole section you wrote one tab back, so its formatted like the rest of the file. Thank you.